manifests/server/extension.pp

```diff
@@ -1,6 +1,7 @@
 # Activate an extension on a postgresql database
 define postgresql::server::extension (
   $database,
+  $extension = $name,
```

Comment:
Why a new parameter for this? It doesn't make sense why we would add a parameter.